website/content/docs/secrets/databases/mysql-maria.mdx

```diff
@@ -166,3 +166,56 @@ API](/vault/api-docs/secret/databases/mysql-maria) page.
 
 For more information on the database secrets engine's HTTP API please see the
 [Database secrets engine API](/vault/api-docs/secret/databases) page.
+
+## Authenticating to Cloud DBs via IAM
+
+### Google Cloud
+
+Aside from IAM roles denoted by [Google's CloudSQL documentation](https://cloud.google.com/sql/docs/postgres/add-manage-iam-users#creating-a-database-user),
+the following SQL privileges are needed by the service account's DB user for minimum functionality with Vault.
+Additional privileges may be needed depending on the SQL configured on the database roles.
+
+```sql
+-- Enable service account to create roles within DB
+GRANT CREATEROLE ON <database>.<object> TO "test-user"@"%";
+```
+
+### Setup
+
+1.  Enable the database secrets engine if it is not already enabled:
+
+    ```shell-session
+    $ vault secrets enable database
+    Success! Enabled the database secrets engine at: database/
+    ```
+
+    By default, the secrets engine will enable at the name of the engine. To
+    enable the secrets engine at a different path, use the `-path` argument.
+
+1.  Configure Vault with the proper plugin and connection information. Here we can explicitly enable GCP IAM authentication
+    and use [Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc#how-to) to authenticate.
+
+    ~> **Note**: For Google Cloud IAM, the Protocol is `cloudsql-mysql` instead of `tcp`.
+
+    ```shell-session
+    $ vault write database/config/my-mysql-database \
+        plugin_name="mysql-database-plugin" \
+        allowed_roles="my-role" \
+        connection_url="user@cloudsql-mysql(project:region:instance)/mysql" \
+        auth_type="gcp_iam"
+    ```
+
+1.  We can also configure the connection and authenticate by directly passing in the service account credentials
```

Comment:
Since this provides additional configuration options, I would make this as a part of the Step 2.  Also, use "you" instead of "we" to clarify who must perform the action.

```suggestion
   You can also configure the connection and authenticate by directly passing in the service account credentials
```